CW-B-W/DataFabric
# add oracle and modify some items

Change items:

- `preview_table_mysql` func can display `datetime` type
- `MetadataScanner.py` can display except information
- oracle can be displayed in `table_preview` and `data_integration` rest api

test:

1. use `MetadataScanner` and `UserManager` func
2. use some rest api:
    oracle:
    http://localhost:5000/table_preview?table_id=111&limit=2
    [{"id": 6, "name": "ted6oracle", "datetest": "2018-06-28T07:30:20"}, {"id": 8, "name": "ted8oracle", "datetest": "2018-06-30T07:30:20"}]

    mysql:
    http://localhost:5000/table_preview?table_id=113&limit=2
    [{"ID": 1, "Name": "ted1", "DateTest": "2022-05-18T13:00:23"}, {"ID": 2, "Name": "ted2", "DateTest": "2022-05-18T13:00:29"}]
   
   `data_integration` check user and column information:
   http://localhost:5000/data_integration?table_ids=111,113
